component/org.wso2.carbon.identity.oauth2.grant.token.exchange/src/main/java/org/wso2/carbon/identity/oauth2/grant/token/exchange/TokenExchangeGrantHandler.java

```diff
@@ -572,6 +582,34 @@ protected String resolveSubject(JWTClaimsSet claimsSet) {
         return claimsSet.getSubject();
     }
 
+    /**
+     * Resolve subject user resident organization value.
+     * @param claimsSet all the JWT claims.
+     *
+     * @return The organization of the subject user resides.
+     */
+    protected String resolveUserResideOrgId(JWTClaimsSet claimsSet) {
```

Comment:
why this protected rather than private

Reply:
Addressed with 21a36df7ce6d42da1a76900a50e19d4ecaa55008